### PR TITLE
Upgrade torch

### LINKED ISSUE #81: Update to torch
Update to torch 0.13.0 and torchdata 0.5.0

its currently pinned in the requirements

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

upgrade torch
use OCF zipper
add some more debug statements

Fixes #81 

## How Has This Been Tested?

CI tests

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: ocf_datapipes/training/gsp_pv_satellite_nwp.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def gsp_pv_nwp_satellite_data_pipeline(
 
     # Load PV data
     logger.debug("Load PV data")
-    pv_datapipe = OpenPVFromNetCDF(pv=configuration.input_data.pv).pv_fill_night_nans()
+    pv_datapipe = OpenPVFromNetCDF(pv=configuration.input_data.pv)  # .pv_fill_night_nans()
```
Comment (peterdudfield) [inline ocf_datapipes/training/gsp_pv_satellite_nwp.py:54]: put this back in

File: ocf_datapipes/batch/merge_numpy_modalities.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 """Merge multiple modalities together in NumpyBatch"""
 from torchdata.datapipes import functional_datapipe
-from torchdata.datapipes.iter import IterDataPipe, Zipper
+from torchdata.datapipes.iter import IterDataPipe
 
 from ocf_datapipes.utils.consts import NumpyBatch
+from ocf_datapipes.utils.utils import ZipperIterDataPipe as Zipper
```
Comment (jacobbieker) [inline ocf_datapipes/batch/merge_numpy_modalities.py:6]: Could you potentially put this in either `ocf_datapipes.__init__.py` for the init file for utils? So then we can do `from ocf_datapipes import Zipper` instead? To match more of the other datapipes.

**Review by jacobbieker — APPROVED** [review]
LGTM! One small thing on moving the import, but thanks a lot!

File: ocf_datapipes/batch/merge_numpy_modalities.py (hunk 2 above)
Comment (peterdudfield) [reply ocf_datapipes/batch/merge_numpy_modalities.py:6]: yea thats a good idea

Comment (peterdudfield) [reply ocf_datapipes/batch/merge_numpy_modalities.py:6]: the others are `from ocf_datapipes.select import DropGSP` so ill go with from `ocf_datapipes.utils import Zipper`, if htats ok?

**Pull request merged**